bin/www

```diff
@@ -25,7 +43,9 @@ var server = http.createServer(app);
  * Listen on provided port, on all network interfaces.
  */
 
-server.listen(port);
+app.loginSuperUser(function () {
+  server.listen(port);
+});
 server.on('error', onError);
```

Comment:
move this above app.login, was a bug before, if listen error sync it will not get called.